### PR TITLE
Productspage

### PR DESCRIPTION
**Pull request by Chiragmodi01**
image

### CODE REVIEW

File: Styles/utils.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.hover-fill {
+    background-color: transparent;
+    box-shadow: inset 0 0px 0 0 black;
+    transition: all .4s;
+}
+.hover-fill:hover {
+    color: white;
+    box-shadow: inset 0 -50px 0 0 black;
```
Comment (haarsh24) [inline Styles/utils.css:8]: Declare colors in root and then use it.

File: Pages/ProductsPage.html [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+                <div class="FilterColor">
+                    <div class="filters_eachFilter filter-color">
+                        <input type="checkbox" id="filter-color color-1" />
+                        <label htmlFor="color-1"></label>
+                        <input type="checkbox" id="filter-color color-2" />
+                        <label htmlFor="color-2"></label>
+                        <input type="checkbox" id="filter-color color-3" />
+                        <label htmlFor="color-3"></label>
+                    </div>
+                </div>
```
Comment (haarsh24) [inline Pages/ProductsPage.html:133]: remove commented code if not needed

**Review by haarsh24 — APPROVED** [review]
Declare colors in root before using them.
Rest all looks good , you can merge.

**Review by Balakumar3018 — APPROVED** [review]
Everything good to go

**Pull request merged**